### PR TITLE
Backports 0.18 pr9

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: doc/psbt.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# PSBT Howto for Bitcoin Core
+
+Since Bitcoin Core 0.17, an RPC interface exists for Partially Signed Bitcoin
```
Comment (UdjinM6) [inline doc/psbt.md:3]: 13941: dashify

File: test/functional/feature_config_args.py [hunk 2]
```diff
@@ -14,6 +14,24 @@ def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 1
 
+    def test_config_file_parser(self):
+        # Assume node is stopped
+
+        inc_conf_file_path = os.path.join(self.nodes[0].datadir, 'include.conf')
+        with open(os.path.join(self.nodes[0].datadir, 'bitcoin.conf'), 'a', encoding='utf-8') as conf:
```
Comment (UdjinM6) [inline test/functional/feature_config_args.py:21]: 14105: dash.conf

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
13249: missing changes in `src/test/test_dash.cpp`

File: doc/psbt.md (hunk 1 above)
Comment (PastaPastaPasta) [reply doc/psbt.md:3]: Should we? or should we just leave this as is for now? How to dashify this is confusing to me

Comment (UdjinM6) [reply doc/psbt.md:3]: Hmmm... why don't we just skip 13941 then?

Comment (PastaPastaPasta) [reply doc/psbt.md:3]: No... I don't think that's needed, because we are supporting psbt at this point..

**Comment by PastaPastaPasta** [comment]
> 13249: missing changes in `src/test/test_dash.cpp`

Looks like this is not needed after rebase

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**